mpas_analysis/shared/climatology/climatology.py

```diff
@@ -862,7 +863,10 @@ def _setup_climatology_caching(ds, startYearClimo, endYearClimo,
 
         cacheIndex = len(cacheInfo)
         for year in years:
-            cacheIndices[yearsInDs == year] = cacheIndex
+            for month in monthValues:
+                mask = numpy.logical_and(yearsInDs == year,
+                                         monthsInDs == month)
+                cacheIndices[mask] = cacheIndex
```

Comment:
@xylar, I'm confused here-- why set this to -1 if it isn't used? I don't see any logic below that uses this knowledge unless I'm missing something.